### PR TITLE
Minor edits for iOS

### PR DESCRIPTION
**Pull request by nassosterz-ms**

### CODE REVIEW

File: iOS/PartySample/app/inc/pch.h [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#include <assert.h>
```
Comment (raymondchams) [inline iOS/PartySample/app/inc/pch.h:1]: You don't introduce assert in this PR. Why we need to add this?

**Review by raymondchams — APPROVED** [review]

File: iOS/PartySample/app/inc/pch.h (hunk 1 above)
Comment (nassosterz-ms) [reply iOS/PartySample/app/inc/pch.h:1]: The code was not building without it, unsure what changed, I see also that the Windows pch has it.

Comment (raymondchams) [reply iOS/PartySample/app/inc/pch.h:1]: Then `#include <cassert>` is preferred style in your first commit.

Comment (nassosterz-ms) [reply iOS/PartySample/app/inc/pch.h:1]: Agreed, but the Windows project uses assert.h https://github.com/PlayFab/PlayFabParty/blob/1eb0ad7a0bc77b873626b12d72a359892fb8abc8/win32/PartySample/dll/pch.h#L10 so I kept it for consistency.

**Review by SahilAshar — APPROVED** [review]

**Pull request merged**